test/lib/code_corps/user_task_matcher_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule CodeCorps.UserTaskMatcherTest do
+  use CodeCorps.ModelCase
+
+  import CodeCorps.UserTaskMatcher
+
+  test "can find top x tasks for a user's skill" do
+    coding = insert(:skill, title: "coding")
+    design = insert(:skill, title: "design")
+
+    account_page = insert(:task)
+    settings_page = insert(:task)
+    photoshop = insert(:task)
```

Comment:
These could maybe be rethought a little to tell a clearer story. Like one could be `front_end_task` and one `back_end_task` and one `design_task`. You'd have `coding_skill` and `design_skill` and then match accordingly. Your final test would be 1 and 3 for length instead, using a limit of 1 and 3.